src/app/metadata/models/storage-aliases.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Interface for the storage aliases.
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+export interface storageAliasDecodes {
```

Comment:
actually this type applies to all values returned by the wkvs, so you could call it something like `WellKnownValues` (names of interfaces should start with a capital letter)